x-pack/legacy/plugins/ml/common/types/messages.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface MessageBasics {
```

Comment:
the name of this file and interface is a bit ambiguous IMO. Mainly because it's just in the `types/` folder and so is missing context.
Would `audit_message.ts` and `AuditMessageBase` be better?

Reply:
Good point! Updated in 35ad48d475c32fd3c7800afabfa2c4bb5985ed99